release/release.go

```diff
@@ -0,0 +1,199 @@
+package main
+
+import (
+	"archive/tar"
+	"archive/zip"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/mongodb/mongo-tools/release/platform"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fail("please provide exactly one subcommand name")
+	}
+	cmd := os.Args[1]
+
+	switch cmd {
```

Comment:
I suggest adding a 'default' case that fails

Reply:
done!

Reply:
done